common-model/src/main/java/com/sequenceiq/common/api/type/LoadBalancerStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.common.api.type;
+
+public enum LoadBalancerStatus {
```

Comment:
I don't love the name `LoadBalancerStatus`, because it's not really the status. What about something like `LoadBalancerCreation`?